examples/seal-key/seal-key.go

```diff
@@ -28,14 +29,51 @@ import (
 	"github.com/chrisccoulson/ubuntu-core-fde-utils"
 )
 
+type pathList []string
+
+func (l *pathList) String() string {
+	var builder bytes.Buffer
+	for i, path := range *l {
+		if i > 0 {
+			builder.WriteString(", ")
+		}
+		builder.WriteString(path)
+	}
+	return builder.String()
+}
+
+func (l *pathList) Set(value string) error {
+	*l = append(*l, value)
+	return nil
+}
+
+type file struct {
```

Comment:
I have the feeling that the name `file` is a bit too generic here, but I see you're using it just to hold the file name for the `ReadAll()` operation below.